apps/src/code-studio/pd/form_components_func/FormController.jsx

```diff
@@ -333,6 +333,12 @@ const FormController = props => {
     if (data?.responseJSON?.errors?.form_data) {
       setErrors(data.responseJSON.errors.form_data);
       setErrorHeader(i18n.formErrorsBelow());
+    } else if (data?.status === 409) {
```

Comment:
I'd put this in a constant (maybe just at the top of this file) so it has a descriptive name